### PR TITLE
Feat/datagrid update

### PR DESCRIPTION
**Pull request by neolitec**
UK-17

Staging: http://feat_datagrid_update_staging.ui-kit.ovh/#!/oui-angular/datagrid
Documentation PR: https://github.com/ovh-ux/ovh-ui-kit-documentation/pull/114

### CODE REVIEW

File: packages/oui-datagrid/README.md [hunk 1]
```diff
@@ -428,6 +428,82 @@ You can use `row-loader`. It take the current row as argument and must return a
 </oui-datagrid>
 ```
 
+### Refresh
+
+If you want to refresh the datagrid, you will have to use a service.
```
Comment (JDownloader) [inline packages/oui-datagrid/README.md]: I would avoid too much conditional, maybe `To refresh a datagrid, you need to use the angular datagrid service` or `The data inside a datagrid can be modified after his initialisation with the angular datagrid service` could do the work

Comment (neolitec) [reply packages/oui-datagrid/README.md]: Hum, yes the doc could be better...

Comment (neolitec) [reply packages/oui-datagrid/README.md]: Like that.

File: packages/oui-datagrid/README.md [hunk 2]
```diff
@@ -430,10 +430,12 @@ You can use `row-loader`. It take the current row as argument and must return a
 
 ### Refresh
 
-If you want to refresh the datagrid, you will have to use a service.
-
 #### Local datagrid
 
+Local datagrid can be refresh by simply mutate `rows` value.
+
+To refresh asynchronous data, you need to define an id for the datagrid and use `ouiDatagridService` to refresh it.
```
Comment (JDownloader) [inline packages/oui-datagrid/README.md]: 👍

**Pull request merged**